<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>159</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appointment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="descriptionBox">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Disable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Note:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="notesBox"/>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QPushButton" name="vacateButton">
        <property name="text">
         <string>Vacate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLineEdit" name="personLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Assignee:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>disableAppointment(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vacateButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vacate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notesBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>changedNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>disableAppointment(bool)</slot>
  <slot>appointmentNote(QString)</slot>
  <slot>vacate()</slot>
  <slot>changedNote()</slot>
 </slots>
</ui>
